app/models/user.rb

```diff
@@ -0,0 +1,15 @@
+class User < ApplicationRecord
+
+    has_many :events, foreign_key: "creator_id", class_name: "Event"
+    has_many :attended_events, foreign_key: "attendee_id"
+    has_many :event_attendance, through: :attended_events, source: :attended_event
```

Comment:
Normally we don't use `source` for the `through` association because it was already obvious which column  you are pertaining to (see line 4 above)